assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -48,11 +52,10 @@ const EditControls = styled.div`
   flex-direction: column;
   justify-content: space-between;
   padding: 0;
-  opacity: 0;
   transition: opacity ease-in-out 300ms;
   background: ${({ theme }) => theme.cardItem.previewOverlay};
   border-radius: ${({ theme }) => theme.storyPreview.borderRadius}px;
-
+  opacity: ${(props) => (props.isActive ? 1 : 0)};
```

Comment:
lol I think I'm the only person who prefers not to destructure just cus I think it reads better.

but just personal preference and habit. happy to change to destructuring cus it's more common in the repo

Reply:
Haha, yeah, this repo definitely takes a destructure first approach.

Reply:
Question, though, why does `opacity` need to be controlled by `isActive` anyway?  It seems to work as intended by letting the css adjust the opacity on hover like it did before? 🤔

Reply:
And if the goal is to simply have it all powered by the state of the card instead of when it's hovered on, then do we need the `&:hover { opacity: 1; }` code?  Should that be removed?

Reply:
Ah yea the `&:hover` can probably be removed.

I made the opacity controlled by `isActive` because the preview should show up if any of it's children have focus as well as hover